openshift.spec

```diff
@@ -2,13 +2,13 @@
 %global debug_package %{nil}
 %global gopath      %{_datadir}/gocode
 %global import_path github.com/openshift/origin
-%global commit      d3f40eafae8ae7bbca61981e33f384375307fafa
+%global commit      21fb40637c4e3507cca1fcab6c4d56b06950a149
```

Comment:
Are we having to maintain this commit manually?  If so I wonder if we could get tito to insert it.  For example, instead of using the shortcommit to set the Kubernetes and OpenShift version we could simply use some combination of the RPM version and release.  A tito --test build should automatically inject the commit hash in to the specfile so perhaps we could use that.  For normal RPM release builds it might be nice to have the version reported by the services to match what the RPM version/release.  Definitely something to test out.